components/basic-profile/edit-profile/grid/wrapper.tsx

```
@@ -0,0 +1,40 @@
+import { useTheme } from "@react-navigation/native";
+import { Platform, ScrollView } from "react-native";
+import { stylesFn } from "@/styles/edit-profile/EditProfile.styles";
+
+interface WrapperProps extends React.PropsWithChildren<{}> { }
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)